java-core/google-cloud-core/src/test/java/com/google/cloud/ServiceOptionsTest.java

```diff
@@ -140,13 +144,44 @@ public class ServiceOptionsTest {
           + "  \"client_email\": \"someclientid@developer.gserviceaccount.com\",\n"
           + "  \"client_id\": \"someclientid.apps.googleusercontent.com\",\n"
           + "  \"type\": \"service_account\",\n"
-          + "  \"quota_project_id\": \"some-quota-project-id\"\n"
+          + "  \"quota_project_id\": \"some-quota-project-id\",\n"
+          + "  \"universe_domain\": \"googleapis.com\"\n"
+          + "}";
+
+  private static final String JSON_KEY_NON_GDU =
```

Comment:
Would you add source code comment how you generated this value?

Reply:
I didn't generate this value. I copied it from above and added a new json entry for `universe_domain`. This `universe_domain` addition to Credentials was part of Auth v1.21.0

Reply:
Can you add that "copied it from above and added a new json entry for universe_domain"?

Reply:
Added